packages/@react-spectrum/checkbox/docs/Checkbox.mdx

```diff
@@ -111,7 +111,7 @@ function Example() {
 
 Checkboxes can display a validation state to communicate to the user if the current value is invalid.
 Implement your own validation logic in your app and pass `"invalid"` to the Checkbox via the `validationState` prop.
-
+The invalid state is only communicated via a color change so in order to be accessible the validation error must also be communicated in another way.
```

Comment:
Given that this is React Spectrum, we should probably give an example of how to communicate this. I looked to see if Spectrum had a suggestion, but I didn't find anything https://spectrum.adobe.com/page/checkbox/#Error

Reply:
Team talked with design, please ignore my comment here for now. We'll followup on the issue.